### PR TITLE
Some pep8 and grep sanity for check_cron_runtime

### PR DESCRIPTION
**Pull request by seqizz**
My project's name includes "grep" word, so `grep -v grep` is not good :smile:

### CODE REVIEW

File: src/check_cron_runtime.py [hunk 1]
```diff
@@ -53,26 +54,27 @@ def parse_args():
...
 
+
 def main(verbose=False):
     """The main program"""
 
     # Dictionary of pids containing th command that is running
     ps_aux = get_ps_aux()
 
-    #pids = psutil.pids()
+    # pids = psutil.pids()
```
Comment (TheFRedFox) [inline src/check_cron_runtime.py]: Looks like old obsolete code. Can't we just remove the line?

File: src/check_cron_runtime.py [hunk 2]
```diff
@@ -101,18 +103,20 @@ def main(verbose=False):
...
     cron_pid_more_than1 = []
-    ok_flag = warn_flag = un_flag = 0
+    ok_flag = warn_flag = 0
     for pid in cron_child_list:
         if pid:
             try:
-                cron_child_time = execute("ps -o 'etimes=' " + pid).strip('    ').strip('\n')
+                cron_child_time = execute(
+                    "ps -o 'etimes=' " + pid
+                ).strip('    ').strip('\n')
```
Comment (TheFRedFox) [inline src/check_cron_runtime.py]: Personal suggestion as the `execute` does some process stuff and I recommend to do this stuff separated, so you can easier check if there is an error during runtime:
``` python
cron_child_time = execute(...)
cron_child_time = cron_child_time.strip(...).strip(...)
```

File: src/check_cron_runtime.py [hunk 3]
```diff
@@ -152,13 +157,17 @@ def get_ps_aux():
...
     return pids
 
+
 def get_child(ppid):
     childs = execute('pgrep -P ' + ppid)
     if childs:
         for child in childs.strip('\n').split():
-       	    print '|' + (execute('ps -ef | grep ' + child + ' | grep -v grep'))
+            print(
+                '|' + (execute('ps -ef | grep ' + child + ' | grep -wv grep'))
```
Comment (TheFRedFox) [inline src/check_cron_runtime.py]: To make it more readable, please store the `execute` result first and also use more `str.format`.

**Comment by TheFRedFox** [comment]
And at line 141 there is a python 2 `print`. So add some braces to make it a method call for python 3 support. 😉

**Comment by seqizz** [comment]
@TheFRedFox Please check again, I force pushed with requested changes. But seems like Github UI does not care :man_facepalming:

File: src/check_cron_runtime.py [hunk 4]
```diff
@@ -152,13 +156,20 @@ def get_ps_aux():
...
 def get_child(ppid):
     childs = execute('pgrep -P ' + ppid)
     if childs:
         for child in childs.strip('\n').split():
-       	    print '|' + (execute('ps -ef | grep ' + child + ' | grep -v grep'))
+            exec_out = execute(
+                    "ps -ef | grep {} | grep -wv grep".format(child)
+                )
+            print(
+                '|' + exec_out
```
Comment (TheFRedFox) [inline src/check_cron_runtime.py]: Can be in one line then.

File: src/check_cron_runtime.py [hunk 5]
```diff
@@ -101,18 +101,21 @@ def main(verbose=False):
...
 
     cron_pid_more_than1 = []
-    ok_flag = warn_flag = un_flag = 0
+    ok_flag = warn_flag = 0
     for pid in cron_child_list:
         if pid:
             try:
-                cron_child_time = execute("ps -o 'etimes=' " + pid).strip('    ').strip('\n')
+                cron_child_time = execute(
+                    "ps -o 'etimes=' " + pid
```
Comment (TheFRedFox) [inline src/check_cron_runtime.py]: Can be in one line then.

File: src/check_cron_runtime.py [hunk 6]
```diff
@@ -131,18 +134,19 @@ def main(verbose=False):
                 print('UNKNOWN PID execution TIME')
                 exit(int(un_code))
 
-    #print cron_pid_more_than1
+    # print cron_pid_more_than1
```
Comment (TheFRedFox) [inline src/check_cron_runtime.py]: Just saw that, seems also obsolete code.

**Pull request merged**